### PR TITLE
[Add] Catalan ASR NGC Resource 

### PR DESCRIPTION
**Pull request by stevehuang52**
Signed-off-by: stevehuang52 <heh@nvidia.com>

The PR adds Catalan ASR models on NGC to the available pretrained models.

### CODE REVIEW

File: nemo/collections/asr/models/rnnt_bpe_models.py [hunk 1]
```diff
@@ -183,6 +183,13 @@ def list_available_models(cls) -> List[PretrainedModelInfo]:
         )
         results.append(model)
 
+        model = PretrainedModelInfo(
+            pretrained_model_name="stt_ca_conformer_transducer_large",
+            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_ca_conformer_transducer_large",
```
Comment (titu1994) [inline nemo/collections/asr/models/rnnt_bpe_models.py:188]: Can you update the docs with these scores for Catalan language ?

Comment (titu1994) [reply nemo/collections/asr/models/rnnt_bpe_models.py:188]: https://github.com/NVIDIA/NeMo/tree/main/docs/source/asr/data and https://github.com/NVIDIA/NeMo/blob/main/docs/source/asr/results.rst

Comment (stevehuang52) [reply nemo/collections/asr/models/rnnt_bpe_models.py:188]: Just added~

Comment (titu1994) [reply nemo/collections/asr/models/rnnt_bpe_models.py:188]: Scores are auto generated. Anyway it's fine I'll change if later

**Review by titu1994 — APPROVED** [review]

**Pull request merged**